spec/controllers/api/group_assignment_repos_controller_spec.rb

```diff
@@ -7,7 +7,8 @@
   let(:user)              { classroom_teacher }
   let(:student)      { classroom_student }
   let(:grouping)     { create(:grouping, organization: organization) }
-  let(:group) { create(:group, grouping: grouping, github_team_id: 3_284_880) }
+  let(:github_team_id) { organization.github_organization.create_team(Faker::Team.name).id }
+  let(:group) { create(:group, grouping: grouping, github_team_id: github_team_id) }
```

Comment:
FYI: I'm working on splitting this test fix in a different PR that will fix this issue in all of our specs. It keeps coming up so I'm going to eliminate it completely in another PR.